.github/CODEOWNERS

```diff
@@ -136,12 +157,56 @@ src/applications/mhv/secure-messaging @department-of-veterans-affairs/vfs-mhv-se
 src/applications/mhv/landing-page @department-of-veterans-affairs/vfs-mhv-integration @department-of-veterans-affairs/va-cto-health-products
 src/applications/avs @department-of-veterans-affairs/va-cto-health-products
 
-# VA.gov Product Teams - Forms
+# Dependents Benefits
 
-src/applications/simple-forms @department-of-veterans-affairs/platform-va-product-forms
```

Comment:
This was removed completely - should it be retained?

Reply:
Yes! Thanks for the callout. Moved it under Platform Design System